src/components/ContentHighlights/HighlightStepper/HighlightStepperConfirmContent.jsx

```diff
@@ -104,10 +130,11 @@ const SelectedContent = () => {
       filterString += ')';
     }
     return filterString;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
[curious] Why is it necessary to disable this ESLint rule? Generally a code smell.